src/napari_matplotlib/tests/test_scatter.py

```diff
@@ -76,7 +76,7 @@ def test_get_valid_axis_keys(make_napari_viewer):
 
 
 def test_get_valid_axis_keys_no_valid_keys(make_napari_viewer):
-    """test the values returned from
+    """Test the values returned from
```

Comment:
Smallest most picky bike-shedding-est comment in the history of the universe, but we have a few...
```python
def foo(bar: str):
    """
    Make a foo.
    """
```
which could(should) be one-liners.

And a few
```python
def foo(bar: str):
    """Short explanation of a foo.

    More details about making a foo.

    Returns
    -------
    foo: a foo.
    """
```

The PEP of "interest" is [257](https://peps.python.org/pep-0257/#multi-line-docstrings). 

Seems like consistency between one-liners and multi liners separated (or not) from `"""` should be a linter-solved problem.

Reply:
Feel free to ignore that an move on with your life.